src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointClusterer.java

```diff
@@ -1,61 +1,70 @@
 package org.broadinstitute.hellbender.tools.spark.sv;
 
+import org.apache.spark.api.java.function.FlatMapFunction;
+import org.apache.spark.api.java.function.Function;
+
 import java.util.*;
-import java.util.function.Function;
 
 /**
  * A class that acts as a filter for breakpoint evidence.
  * It passes only that evidence that is part of a putative cluster.
  */
-public final class BreakpointClusterer implements Function<BreakpointEvidence, Iterator<BreakpointEvidence>> {
+public final class BreakpointClusterer implements Iterator<BreakpointEvidence> {
     private final int minEvidenceCount;
     private final int staleEventDistance;
```

Comment:
Sure I worried about it, but I think it's ultimately a lot less data than the kmerization, so now I'm not as worried.  And it's so much simpler this way.  There are on the order of 10^6 reads per partition, a BreakpointEvidence object is on the order of 10^3 bytes, so that's only 10^9 bytes even if every read were evidence.  Don't think it'll blow up.
I've removed staleEventDistance to clean up.